recipes/send2trash/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "Send2Trash" %}
+{% set version = "1.4.2" %}
+{% set sha256 = "725fbce571dffe0b640e2f1788d52c3c544b510f9d8f69b2597c8c2555bc8441" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - python
```

Comment:
I don't think you need to list `python` in the `test:requires` section since the testing environment should get all of the `run` reqs.

Reply:
removed!